website/client/js/controllers/inviteToGroupCtrl.js

```diff
@@ -44,8 +44,13 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       Groups.Group.invite($scope.group._id, invitationDetails)
         .then(function() {
-          Notification.text(window.env.t('invitationsSent'));
-          _resetInvitees();
+           if ((inviteMethod === 'email' && invitationDetails.emails.length > 1) || (inviteMethod === 'uuid' && invitationDetails.uuids.length > 1)) { // test if it's inviting more than 1 emails or uuid
+                Notification.text(window.env.t('invitationsSent'));
+            }
+            else {
+                Notification.text(window.env.t('invitationSent'));
+            }
+            _resetInvitees();
```

Comment:
@khipkin I'm so sorry! This whole time I thought you were talking about the `inviteToGroupCtrl.js` file, I didn't see the spec at the end. 
Ok, so I checked it out, and it seems that the multiple uuid invites part is already covered (unless I'm mistaken). But I still couldn't figure out how to test the changes I made to the `inviteToGroupCtrl.js` file. All I did here was change the string of the notification, I couldn't find anything like that in the `inviteToGroupCtrlSpec.js`. I'm sorry :(